Templates/assignTA.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
I may have missed it during the meeting today, but what is the intended purpose of the assignTA page? Is it for instructors or supervisors?

Reply:
I just wrote a message in teams for what Yazan and I believe to be the best course of action for assigning TAs. I will be editing this html page to act as a view page so we can ignore it for now as it doesn't work fully/ doesn't break anything.